### PR TITLE
add line-clamp-<n> atomic class

### PR DESCRIPTION
**Pull request by wibjorn**
Task: task-516180

Link: preview-281 and visit atomics/line-clamp

### CODE REVIEW

File: site/src/scaffold/styles/layout.scss [hunk 1]
```diff
@@ -102,9 +102,11 @@ $size-bar-width: 300px - strip-units($gap-size) * $document-font-size;
 
 #article {
 	grid-area: article;
+	max-width: 100%;
```
Comment (wibjorn) [inline site/src/scaffold/styles/layout.scss:105]: Notice this is required to ensure the main content area does not overflow.

Comment (jdanyow) [reply site/src/scaffold/styles/layout.scss:105]: seems to work without it- specific resolution needed?

File: css/src/components/truncated.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	-webkit-box-orient: vertical;
+	overflow: hidden;
+}
+
+@mixin truncated($lines: 1) {
+	@extend %truncated;
+	-webkit-line-clamp: $lines !important;
+}
+
+.truncated-1 {
```
Comment (jdanyow) [inline css/src/components/truncated.scss:13]: .line-clamp-1

Comment (wibjorn) [reply css/src/components/truncated.scss]: updated

**Review by jdanyow — DISMISSED** [review]

**Review by ollips — APPROVED** [review]

**Review by ollips — APPROVED** [review]

**Review by jdanyow — APPROVED** [review]

**Pull request merged**